internal/service/sso_jwks.go

```
@@ -0,0 +1,185 @@
+
+func (s *SSOService) getOAuthProviderDocument(ctx context.Context, uri string) ([]byte, error) {
+	request, err := http.NewRequestWithContext(ctx, http.MethodGet, uri, nil)
+	if err != nil {
+		return nil, ErrOAuthProviderUnavailable
+	}
+	request.Header.Set("Accept", "application/json")
+	response, err := s.httpClient.Do(request)
+	if err != nil {
+		return nil, ErrOAuthProviderUnavailable
+	}
+	defer response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)